libr/core/cmd_anal.inc.c

```diff
@@ -1454,7 +1454,7 @@ static bool cmd_anal_aaft(RCore *core) {
 		}
 		r_reg_arena_poke (core->anal->reg, saved_arena, saved_arena_size);
 		r_esil_set_pc (core->anal->esil, fcn->addr);
-		r_core_anal_type_match (core, fcn);
+		r_anal_type_match (core->anal, fcn);
```

Comment:
as long as this api is in a plugin now, we should just call the command. you can do : r_core_cmd0(core, "a:tp"); (or something like that